src/index.ts

```diff
@@ -72,6 +72,44 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  //handle push event
+  app.on("push", async (context) => {
+    kc.loadFromDefault();
+    const client = kc.makeApiClient(k8s.CustomObjectsApi);
```

Comment:
Please create this client at global scope

Reply:
Resolved in latest commit.